mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/137-Component User tab

Closes #137 

To test:
- Run Starting Scenarios/TirePly Filter Scenario scenario
- Open Component View and click on User Groups
- Select one of the computers (not the servers) and make sure appropriate accounts are highlighted
- Assign an authenticated server and make sure accounts section goes away
- Enable/disable profiles and make sure they stay correct when selecting other computers
- Click a server and make sure authenticating server tab goes away
